### PR TITLE
#166722761 Correct automation stats

### PR DESCRIPTION
**Pull request by johnkegz**
#### What does this PR do?
- This PR fixes the automation status on developer card.
##### actual
 - automation status displays success/failure
##### expected
 - automation status should be success/total
#### How should this be manually tested?
- clone the backend repo and the frontend repo `git clone https://github.com/andela/bp-esa-frontend.git` and `git clone https://github.com/andela/bp-esa-backend.git` respectively.
- install dependencies `yarn`
- Login into the application and click on the card view option.
- Automation status will be success/total
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#166722761](https://www.pivotaltracker.com/story/show/166722761)
#### Postman Documentation
- N/A
#### Screenshots (If applicable)
Before (success/failure)
Screen Shot 2019-06-18 at 12 05 21

After (success/total)
Screen Shot 2019-06-17 at 12 12 58

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```
Comment (MuhanguziDavid) [inline .gitignore]: Nice work @johnkegz , however, why did you add `yarn.lock` to `.gitignore`

Comment (johnkegz) [reply .gitignore]: I had put it there because I thought some people use npm which generates `package-lock.json` file but anyway let me remove it from there.

**Review by MuhanguziDavid — APPROVED** [review]
